terraform/Using_Terraform.md

```diff
@@ -13,20 +13,20 @@ The Makefile contain all the commands to use the Terraform, however if you'd lik
 3. Create the file `my_variables.auto.tfvars` with the following Terraform input variables using your own specific values:
 
    ```hcl
-   owner                        = "johandry"
+   owner                        = "bob"
    project_name                 = "cloud-pak-app"
-   entitled_registry_user_email = "Johandry.Amador@ibm.com"
+   entitled_registry_user_email = "bob@email.com"
    ```
 
-   Append the input variable `cluster_id` if you have an existing Openshift cluster to install the Cloud Pak, like so:
+   Append the input variable `cluster_id` if you have an existing Openshift cluster to install the Cloud Pak on, like so:
 
    ```hcl
-   cluster_id                   = "bupkhgbd00id2uu15cjg"
+   cluster_id                   = "xxxxxxxxxxxxxxxxxxxxx"
    ```
 
-   Open the file `terraform.tfvars` or `variables.tf` and feel free to overwrite the input default values for any of the existing variables.
+   Open the file `cp4xxx/terraform.tfvars` or `variables.tf` to verify or overwrite the input default values for any of the existing variables. For more information on each Cloud Pak's specific inputs go to [cp4mcm](cp4mcm/README.md), [cp4app](cp4app/README.md) or [cp4data](cp4data/README.md).
```

Comment:
The user is already in the `cp4xxx` directory, so I think it's fine to have just the filename.